api-gateway/src/main/resources/application.yml

```diff
@@ -60,6 +62,14 @@ spring:
             # If SESSION cookie is passed to downstream, it is then removed, because downstream discards it
             - RemoveRequestHeader=Cookie
             - ConvertAuthorizationHeader=
+        - id: demo-api_route
+          uri: ${gateway.demo.url}
+          predicates:
+            - Path=/demo/api/**
+          filters:
+            # If SESSION cookie is passed to downstream, it is then removed, because downstream discards it
+            - RemoveRequestHeader=Cookie
+            - ConvertAuthorizationHeader=
```

Comment:
Do we need to pass authentication to demo? Or will it be useful in the future?

Reply:
I guess it might be useful as far as we might want to store some user files on save-demo

Reply:
big brother is watching you